### PR TITLE
adds change notifications delivery alternative

### PR DESCRIPTION
**Pull request by baywet**

### CODE REVIEW

**Comment by baywet** [comment]
@davidmu1 thanks for taking the time to review this new preview article for subscriptions.
We probably need to refer it in the main concept article and maybe in post beta subscriptions. What do you think?

File: concepts/change-notifications-delivery.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+# Get change notifications delivered different ways (preview)
+
+Change notifications can be delivered different ways to subscribers. If change notifications' main delivery mode is through webhooks, it can be challenging to leverage webhooks for high throughput scenarios.
+
+## Using Azure Event Hubs to receive change notifications
+
+[Azure Event Hubs](https://azure.microsoft.com/services/event-hubs) is a popular real-time events ingestion and distribution service built for scale. You can leverage Azure Events Hubs to receive change notifications instead of traditional webhooks. This feature is currently in preview.  
+Using Azure Event Hubs to receive change notifications differs in a few ways including:
+
+- You don't rely on publicly exposed notification URLs : the Event Hubs SDK will relay the notifications to yoru application
```
Comment (venkateshchepuru) [inline concepts/change-notifications-delivery.md]: Typo : yoru -> your

Comment (baywet) [reply concepts/change-notifications-delivery.md]: thanks! fixed

File: concepts/change-notifications-delivery.md [hunk 2]
```diff
@@ -0,0 +1,189 @@
...
+
+The main difference during subscription creation will be the **notificationUrl**. You must set it to `EventHub:https://<azurekeyvaultname>.vault.azure.net/secrets/<secretname>?tenantId=<domainname>`, with the following values:
+
+- `azurekeyvaultname` - The name you gave to the key vault when you created it. Can be found in the DNS name.
+- `secretname` - The name you gave to the secret when you created it. Can be found on the Azure Key Vault **Secrets** page.
+- `domainname` - The name of your tenant; for example, consto.onmicrosoft.com or contoso.com. Because this domain will be used to access the Azure Key Vault, it is important that it matches the domain used by the Azure subscription that holds the Azure Key Vault. To get this information, you can go to the overview page of the Azure Key Vault you created and click the subscription. The domain name is displayed under the **Directory** field.
+
+## Next steps
```
Comment (Lauragra) [inline concepts/change-notifications-delivery.md:147]: Should this Next steps section be at the end of the article? This section isn't following the heading level hierarchy.

File: concepts/change-notifications-delivery.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+
+#### Receiving notifications
+
+Events will be now delivered to your application by Event Hubs. For details, see [receiving events](https://docs.microsoft.com/azure/event-hubs/get-started-dotnet-standard-send-v2#receive-events) in the Event Hubs documentation.
+
+Before you can receive the notifications in your application, you'll need to create another shared access policy with a "Listen" permission and obtain the connection string, similar to the steps listed in [Configuring the Azure Event Hub](#configuring-the-azure-event-hub).
+
+> **Note:** Create a separate policy for the application that listens to Event Hubs messages instead of reusing the same connection string you set in Azure KeyVault. This ensures that each component of the solution has only the permissions it needs and follows the least permissions security principle.
+
+### Frequent questions
```
Comment (Lauragra) [inline concepts/change-notifications-delivery.md:164]: Do you mean "Frequently asked questions"? And should this be an H2? Is there another question that you can add to merit the plural heading? ;)

File: concepts/change-notifications-delivery.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+#### Microsoft Graph change tracking application missing
```
Comment (Lauragra) [inline concepts/change-notifications-delivery.md:166]: Recast the heading as a question? i.e. "What happens if the Microsoft Graph change tracking application is missing?"

Comment (baywet) [reply concepts/change-notifications-delivery.md]: thanks! I when this direction instead of adding another FAQ

**Pull request merged**